charts/dex/templates/deployment.yaml

```diff
@@ -14,10 +14,10 @@ spec:
   template:
     metadata:
       annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
```

Comment:
It would probably be better to check the value of `.Values.configSecret.create` and add the annotation based on that.

Reply:
Right, I will do this

Reply:
Done